tool/tctl/common/auth_command.go

```diff
@@ -503,10 +508,10 @@ func writeHelperMessageDBmTLS(writer io.Writer, filesWritten []string, output st
 		// Consider adding one to ease the installation for the end-user
 		return nil
 	}
-
 	tplVars := map[string]interface{}{
-		"files":  strings.Join(filesWritten, ", "),
-		"output": output,
+		"files":       strings.Join(filesWritten, ", "),
+		"jksPassword": jksPassword,
```

Comment:
Do we want to include `jksPassword` if it is empty?

Reply:
I'd argue that we shouldn't include it if the password is shorter than 8 characters min or even add some rules to force strong passwords here.

Reply:
@smallinsky I had a conversation with @reedloden about it, and we think that generating a password for a user using `utils.CryptoRandomHex()` is a better approach. We can then generate stronger passwords and give users encrypted files and the password that they can.

Reply:
I have switched to `utils.CryptoRandomHex()`